R/generateFilterValues.R

```diff
@@ -110,7 +109,11 @@ generateFilterValuesData = function(task, method = "randomForestSRC_importance",
     } else {
       stopf("You use more than 1 filter method. Please pass extra arguments via 'more.args' and not '...' to filter methods!")
     }
+  } else if (length(method) == 1L && length(more.args) > 1L) { # simple filter with 1 method and more.args
```

Comment:
What if I want to set only parameters that are not `method`?

Reply:
That won't be a problem. "method" here is the argument to generateFilterValuesData, specifying the type of filter to use, not the method argument to a specific filter like randomForestSRC_var.select. So this test is just checking to see if a simple filter has been used (as opposed to an ensemble filter), with arguments passed inline, not as part of a named list, for example:
`fv = generateFilterValuesData(iris.task, method = "randomForestSRC_var.select", nselect = 2, metho="md", nrep=5)`
If that is the case then the arguments are formatted as a named list, named by the filter method.
I hope that makes sense. Am I understanding your concerns correctly?

Reply:
Ok thanks.